stdlib/interval/interval_test.flux

```diff
@@ -38,7 +38,7 @@ package interval_test
 import "experimental"
 import "interval"
 
-// these test cases cannot pass CI, so they're skipped for now. Once we're able to test a non table stream test case,
+// TODO(faith): these test cases cannot pass CI, so they're skipped for now. Once we're able to test a non table stream test case,
```

Comment:
We actually now have the ability to do tests on non stream cases. Ask @rockstar or @jsternberg for direction.  Lets not block this PR on that work. Can you create a new issue to unskip these tests using the new Flux acceptance tests?

Reply:
oh yay! added this issue to the agenda https://github.com/influxdata/flux/issues/3526